keyboards/dichotomy/keymaps/alairock/keymap.c

```diff
@@ -0,0 +1,205 @@
+// this is the style you want to emulate.
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+
+#include "dichotomy.h"
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _MOUSE 8
+#define _ADJUST 16
+
+
+enum dichotomy_keycodes
+{
+	QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+  MOUKEY,
+  MS_BTN1,
+  MS_BTN2,
+  MS_BTN3
+};
+
+#define RED_BRIGHTNESS 3
+#define GREEN_BRIGHTNESS 2
+#define BLUE_BRIGHTNESS 2
+
+// Fillers to make layering more clear
```

Comment:
Lines 33-35 can be deleted; QMK's build script includes these definitions in every keymap.

https://github.com/qmk/qmk_firmware/blob/7c63c0a95075f6fb4dd94d020ceb5816e6bcca2c/quantum/quantum_keycodes.h#L29-L31